packages/checkout/sdk/src/types/smartCheckout.ts

```diff
@@ -345,6 +345,8 @@ export interface SmartCheckoutParams {
   onComplete?: (result: SmartCheckoutResult) => void;
   /** The callback to be executed on each funding route found */
   onFundingRoute?: (fundingRoute: FundingRoute) => void;
+  /** A flag to get funding routes based pm full or partial amonut */
```

Comment:
what does `pm` mean here? also small nit on spelling fix for `amonut`

Reply:
a my lil cooked brain, good catch Charlie!